src/components/WwwFrame/SecondaryMenu.vue

```diff
@@ -29,12 +29,12 @@
 				text-decoration: none;
 
 				&:hover,
-				&.router-link-active {
+				&.router-link-exact-active {
 					color: $kiva-accent-green;
 					border-bottom: 2px solid $kiva-accent-green;
 				}
 
-				&.router-link-active {
+				&.router-link-exact-active {
```

Comment:
@ksekhri We need to keep both of these general (remove the 'exact') in order to preserve the selected styling for the parent of the selected page, ie the Selected Section. See the secondary nav on /portfolio/lending-stats

Reply:
The more practical approach would probably be to just add &.router-link-exact-active to both groups so this style is applied in all 3 scenarios.

Reply:
> The more practical approach would probably be to just add &.router-link-exact-active to both groups so this style is applied in all 3 scenarios.

When I tried that the "Developer Home" heading was highlighted even when the user is navigated to the sub page `/build-vue/docs`

I've come up with something new, let me know if this approach looks good to you @mcstover I've let users of the component decide whether they want the matches to be exact or not based on props.